5.5/Dockerfile

```diff
@@ -34,8 +34,8 @@ ENV MYSQL_VERSION 5.5.52
 
 # note: we're pulling the *.asc file from mysql.he.net instead of dev.mysql.com because the official mirror 404s that file for whatever reason - maybe it's at a different path?
 RUN apt-get update && apt-get install -y wget --no-install-recommends && rm -rf /var/lib/apt/lists/* \
```

Comment:
After one does `apt-get install curl`, an `apt-get purge --auto-remove curl` leaves packages around that `wget` doesn't even install, which is why we usually try to prefer `wget` for these.